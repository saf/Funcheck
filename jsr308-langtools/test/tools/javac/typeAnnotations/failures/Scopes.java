/*
 * @test /nodynamiccopyright/
 * @bug 6843077
 * @summary check that A is accessible in the class type parameters
 * @author Mahmood Ali
 * @compile/fail/ref=Scopes.out -XDrawDiagnostics -source 1.7 Scopes.java
 */
class Scopes<T extends @UniqueInner Object> {
  @interface UniqueInner { };
}
