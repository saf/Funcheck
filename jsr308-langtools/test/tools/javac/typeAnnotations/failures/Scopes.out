Scopes.java:8:25: compiler.err.cant.resolve: kindname.class, UniqueInner, , 
1 error
